cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -606,6 +625,10 @@ def _configure_cluster(self):
                 'If the majority of cluster nodes are not yet installed then '
                 'this warning can be ignored.'
             )
+        except AuthSetupInProgress:
```

Comment:
i'd prefer factoring the above out to a separate function instead of using exceptions for flow control

Reply:
Different approach being used now, closing thread.